packages/zowe-explorer/src/utils/ProfilesUtils.ts

```diff
@@ -734,24 +734,15 @@ export class ProfilesUtils {
         Gui.infoMessage(responseMsg.join(""), { vsCodeOpts: { modal: true } });
     }
 
-    public static extenderTypeReady: Map<string, imperative.DeferredPromise<void>> = new Map([
-        [
-            "zosmf",
-            ((): imperative.DeferredPromise<void> => {
-                const deferred = new imperative.DeferredPromise<void>();
-                deferred.resolve();
-                return deferred;
-            })(),
-        ],
-    ]);
-
-    public static async awaitExtenderType(profileName: string, profInfo: ProfilesCache): Promise<void> {
-        const profAttrs = await profInfo.getProfileFromConfig(profileName);
-        if (profAttrs && !ProfilesUtils.extenderTypeReady.has(profAttrs.profType)) {
+    private static extenderTypeReady: Map<string, imperative.DeferredPromise<void>> = new Map();
```

Comment:
Thoughts on renaming `extenderTypeReady` to `extenderProfileReady`? The map no longer stores deferred promises based on the profile type, so it might be easier for another developer to digest if it was named accordingly